src/main/java/where2eat.ump

```diff
@@ -0,0 +1,68 @@
+namespace ca.mcgill.ecse.where2eat.model;
+
+enum Category {Chinese, Indian, Portuguese}
```

Comment:
We should definitely add more Categories into this enum eventually. Essentially, scrape [what's provided](https://www.yelp.ca/developers/documentation/v3/category_list) (CTRL-F Restaurants) by Yelp's API since we said that's what we would use in tandem with PlacesAPI for restaurant search. Note that Yelp's categories as used by their API are lower-cased, we should just conform 1-to-1 since it reduces the overall complexity, I believe.